src/platform/forms-system/test/js/review/ReviewCollapsibleChapter.unit.spec.jsx

```diff
@@ -170,17 +172,23 @@ describe('<ReviewCollapsibleChapter>', () => {
         showPagePerItem: true,
         arrayPath: 'testing',
         path: 'path/:index',
+        index: 0,
       },
       {
         title: '',
         pageKey: 'test',
         showPagePerItem: true,
         arrayPath: 'testing',
         path: 'path/:index',
+        index: 1,
```

Comment:
I know that `page.index` existed before this PR, but why do we need these now? Would existing configs be considered broken without this?

Reply:
Yes.

Reply:
To be clear, though, this isn't specified in the `formConfig`s. These come from the way the forms library hydrates array routes. I'm unclear on the details, but essentially it duplicates each page config with its own `index` for each item in the array at `arrayPath`.